k8s.go

```diff
@@ -261,7 +261,7 @@ type KubernetesNodePoolProperties struct {
 
 	// Public Ips
 	// Required: false
-	PublicIps	[]string	`json:"publicIps,omitempty"`
+	PublicIPs *[]string `json:"publicIps,omitempty"`
```

Comment:
Why do you need a pointer here even though you can also assign `nil` to `[]string`?

Reply:
Agreed, however, I was trying to follow the general approach for the rest of the structure (see LANs) - thought it was preferred that way. Will revert it, no problem.

Reply:
@piepmatz - done

Reply:
Alright, thx. I do value consistency a lot, but in this case I'd rather go for the better solution than sticking to the pointer approach taken for LANs.